src/services/anchor-service.ts

```diff
@@ -160,7 +160,7 @@ export class AnchorService {
   ) {
     this.merkleDepthLimit = config.merkleDepthLimit
     this.useSmartContractAnchors = config.useSmartContractAnchors
-    this.useQueueBatches = config.queue.sqsQueueUrl !== ''
+    this.useQueueBatches = Boolean(config.queue.sqsQueueUrl)
```

Comment:
I think this is due to javascript's truthyness vs Go?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

Either way, LGTM!

Reply:
I'm actually assuming that if the sqsQueueUrl isn't set, the default config will fill it in with an empty string. Thats why the issues you and Mohsin are seeing are so weird....